src/services/users.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+import Utils from './utils';
+
+
+export async function getAllUsers() {
+  const config = {
+    headers: {
+      'Access-Control-Allow-Origin': '*',
+      Accept: 'application/json',
+    },
+  };
+
+  return await axios
+  .get(`${Utils.getBaseUrl()}/usuarios`, config)
+  .then((respose) => respose)
+  .catch((error) => error)
+}
+
+export async function registerUser(name, email, password, administrador) {
```

Comment:
```suggestion
export async function registerUser({ name, email, password, administrador }) {
```
Fazer o mesmo ajuste fornecido no review do `registerProduct`.